### PR TITLE
Add training github team.

### PR DESCRIPTION
**Pull request by HumairAK**
SSIA

### CODE REVIEW

File: github-config.yaml [hunk 1]
```diff
@@ -419,3 +419,10 @@ orgs:
         members:
           - HumairAK
           - durandom
+      training:
+        description: Manage all training content for users and contributors
+        privacy: closed
+        members:
+          - stefwrite
```
Comment (harshad16) [inline github-config.yaml:426]: The member mentioned , i dont see the person listed a member of operate-first
`https://github.com/operate-first/common/blob/b3325bf886388d6c9282650d2fac9a571317fff1/github-config.yaml#L64-L72`

**Review by harshad16 — CHANGES_REQUESTED** [review]
we should include the `training` repo for admin and traige group. 
Also there is one minor issue

File: github-config.yaml (hunk 1 above)
Comment (HumairAK) [reply github-config.yaml:426]: https://github.com/operate-first/common/blob/main/github-config.yaml#L80

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**